movement_primitives/dmp/_state_following_dmp.py

```diff
@@ -34,8 +34,26 @@ def __init__(self, n_dims, execution_time, dt=0.01, n_viapoints=10,
 
         alpha_z = canonical_system_alpha(0.01, self.execution_time, 0.0)
 
-        self.alpha_y = 25.0
-        self.beta_y = self.alpha_y / 4.0
+        if isinstance(alpha_y, float):
```

Comment:
duplicate code

Reply:
Please see [9103981](https://github.com/dfki-ric/movement_primitives/pull/59/commits/91039818c8624e4c282220596192a4bb0863932a), and let me know if this works for you?